src/content/docs/journey-large-logs/organize.mdx

```diff
@@ -0,0 +1,196 @@
+---
+title: Organize your logs 
+metaDescription: "Test"
+---
+
+import logsParsing from 'images/logs_screenshot_full-parsing.webp'
+import logsPartition from 'images/logs_screenshot_full-partition.webp'
+
+Once you've decided what logs to ingest and store, it's time to organize your logs. You're still likely ingesting hundreds of gigabytes or dozens of terabytes of logs. While it's a lot less than you originally had, you'll still have toil while trying to effectively use them. 
+
+New Relic provides the ability to group these remaining logs into thematic partitions and parse the logs to pull out and tag valuable attributes. By organizing your logs in this manner, you can:
+
+* Query for any attribute within your logs
+* Manage specific partitions at a time, such as front-end logs vs back-end logs
+* Reduce query load times 
+
+
+## Why partition logs
```

Comment:
Suggest adding `your` in here. Since partition is both a verb and a noun, this is a little tough to parse on first read.